#! /bin/sh /usr/share/dpatch/dpatch-run
## it_keymap.dpatch by Patrick Calhoun <phineas@ou.edu>
## (original patch by Luca Tettamanti <kronos.it@gmail.com>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Italian Keyboard map that includes brace characters.

@DPATCH@
diff -urNad rdesktop-1.6.0~/keymaps/it rdesktop-1.6.0/keymaps/it
--- rdesktop-1.6.0~/keymaps/it	2003-11-03 08:54:52.000000000 -0600
+++ rdesktop-1.6.0/keymaps/it	2009-08-19 15:29:22.000000000 -0500
@@ -58,11 +58,11 @@
 egrave 0x1a
 eacute 0x1a shift
 bracketleft 0x1a altgr
-dead_abovering 0x1a shift altgr
+braceleft 0x1a shift altgr
 plus 0x1b
 asterisk 0x1b shift
 bracketright 0x1b altgr
-dead_macron 0x1b shift altgr
+braceright 0x1b shift altgr
 ae 0x1e altgr
 AE 0x1e shift altgr
 ssharp 0x1f altgr
